src/main/java/com/mongodb/kafka/connect/source/StartedMongoSourceTask.java

```diff
@@ -435,6 +455,10 @@ private MongoChangeStreamCursor<? extends BsonDocument> tryCreateCursor(
             e.getErrorCode(),
             e.getErrorMessage());
         throw new ConnectException("Illegal $changeStream operation", e);
+      } else if (e.getErrorCode() == UNKNOWN_FIELD_ERROR) {
+        String msg = format("Invalid operation: %s %s.", e.getErrorMessage(), e.getErrorCode());
```

Comment:
This error message is likely to confuse users - this should only happen on MongoDB < 4.0 with a timestamp set. Can you add more information to help users determine the root cause of issue and how to fix?

Reply:
Done.